src/main/java/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,21 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
```

Comment:
According to the checklist, you should avoid passing the operation type into the method if it's not being used. Since this is a `SupplyOperation`, the operation type is implicit and doesn't need to be a parameter.